pyproject.toml

```diff
@@ -103,9 +100,6 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools]
 zip-safe = false
 include-package-data = true
-license-files = [
-    "LICENSE.rst",
```

Comment:
do we want to keep `LICENSE.rst` in the entry above as well?

Reply:
oh, sorry! I was looking at a different directory and just thought this didn't exist 😅 

I'll add it to the `license-files`, yes